sstables/leveled_compaction_strategy.cc

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) 2019 ScyllaDB
```

Comment:
nit: since this code was merely copied over it should carry the original year it was authored, not this year,.
Otherwise, LGTM

Reply:
It's okay to update the copyright year.